### DISCUSSION TITLE
Best way to style <Link> components using Tailwind

### DISCUSSION THREAD

**Discussion by verheyenkoen** [vercel/next.js#52397] [Help] [answered]
### Summary

With Tailwind, `<Link>` components are unstyled by default. I know I could wrap it inside another component to add generic styling, and pass on the props, but I was wondering if there is no better way to just tell Tailwind (or Next.js): whenever you encouter this component, style it like this?

**Comment by hamidmayeli** [comment, marked as answer]
You can add a section in the `global.css` to style anchors.

```css
a {
    @apply px-3 py-1 rounded-full bg-black dark:bg-slate-200 text-white dark:text-black;
}
```

**Reply by verheyenkoen** [reply]
Oh wow, why didn't I think of that? I knew about `@apply` but for some reason don't think about regular CSS files anymore. Great solution. Thanks!
